torch/onnx/symbolic_helper.py

```diff
@@ -1712,36 +1712,6 @@ def args_have_same_dtype(args):
     return has_same_dtype
 
 
-# TODO(justinchuby): Delete these setters, users should set the vars directly.
-@_deprecation.deprecated(
-    "1.13",
-    "2.0",
-    "remove its usage and avoid setting internal variables directly",
-)
-def _set_opset_version(opset_version: int):
```

Comment:
For all call sites, could you replace the call with `GLOBALS.export_onnx_opset_version = opset_version`? Thanks!